packages/wonder-blocks-icon-button/components/icon-button.md

```diff
@@ -9,7 +9,7 @@ const styles = StyleSheet.create({
         flexDirection: "row",
     },
     sideMargins: {
-        marginRight: 10,
+        marginRight: 16,
```

Comment:
I think we should update the toolbar demos and all the places where iconbuttons are used to use Strut spacing instead of CSS margins. We should have our demo code represent how people should be using the components.

Right now I'm worried that the default state for IconButton is going to end up with people putting iconbuttons next to each other and having the focus rings overlap - so we should be putting the defaults in for whatever the designers want with some Struts. Same goes for buttons too - we should update all the demos to represent this (and it should be part of this PR as the problem is going to become much more pronounced).

Reply:
I'll update the examples.  I doubt people will end up with overlapping focus rings since the focus ring is never shown in any of the mocks, only the icon itself is.